src/thunderbots/software/util/logger/init.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <g3log/g3log.hpp>
+#include <g3log/logworker.hpp>
+
+#include "util/logger/custom_g3log_sinks.h"
+
+namespace Util
+{
+    namespace Logger
+    {
+        /**
+         * Initializes a g3log logger for the calling program. This should only be called
+         * once at the start of a program.
+         */
+        void init_logger()
```

Comment:
Another small nit, but is it worth mentioning that this self-destroys after scope exit? I was quite confused on why there wasn't a deinit function until reading the docs.

Reply:
This is now documented with the LoggerSingleton class